3600/3/Gemfile

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+gem 'dotenv'
```

Comment:
If you do
```ruby
gem 'dotenv', require: 'dotenv/load'
```
`Bundler.require` will require and load dotenv, so won't be necessary `Dotenv.load`

Reply:
Removed

Reply:
Dotenv.load deleted

Reply:
And I don't understand where you got this from: require: 'dotenv/load'

Reply:
From documentation of bundler

Reply:
WIthout `require: 'dotenv/load'` you `.env` file won't be loaded

Reply:
Those I just have to add require: 'dotenv/load' to gem 'dotenv'?